float64 x_pos
float64 y_pos
float64 linear_velocity
float64 linear_acceleration

float64 roll
float64 pitch
float64 yaw
float64 yaw_rate
